Hestia/Hestia/DevicesScreen/TableSource.cs

```diff
@@ -182,7 +174,7 @@ public void WillBeginTableEditing(UITableView tableView)
             // This should not be permanently stored, but trigger the add new
             // device screen on touch
             List<backend.models.Activator> temp_activator = new List<backend.models.Activator>();
-            NetworkHandler temp_networkhandler = new NetworkHandler("94.212.164.28", 8000);
+            NetworkHandler temp_networkhandler = new NetworkHandler(Globals.IP, Globals.Port);
```

Comment:
Thanks, though will you make sure to ask the user for its own input IP first time ?